handlers/prometheus_handlers.go

```diff
@@ -34,8 +34,9 @@ func (h *Handler) ScanPromGrafanaHandler(w http.ResponseWriter, req *http.Reques
 		return
 	}
 	if err = json.NewEncoder(w).Encode(availablePromGrafana); err != nil {
-		h.log.Error(ErrMarshal(err))
-		http.Error(w, ErrMarshal(err).Error(), http.StatusInternalServerError)
+		obj := "payloads"
```

Comment:
shouldn't this be specific? Like prometheus payload or grafana payload?

Reply:
Okay @kumarabd I will update this in my next pr